CMakeLists.txt

```diff
@@ -95,15 +91,30 @@ if (WITH_TITAN_TESTS OR WITH_TITAN_TOOLS)
   add_subdirectory(${ROCKSDB_DIR} rocksdb EXCLUDE_FROM_ALL)
   # -latomic is needed when building titandb_stress using clang.
   # check if libatomic exist before link
-  find_library(HAS_ATOMIC NAMES atomic libatomic.so.1 atomic.so.1)
-  if (HAS_ATOMIC)
-    link_libraries(atomic)
+  # Check if -latomic is required or not
+  if (NOT MSVC)
+    set(CMAKE_REQUIRED_FLAGS "--std=c++11")
+    CHECK_CXX_SOURCE_COMPILES("
+  #include <atomic>
```

Comment:
add indent?

Reply:
I copied it from rocksdb, just keep it consistent.